src/app/search/search.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="main">
+  <section>
+    <header>
+      <h2>Do a search for...</h2>
+
+      <div class="form-group">
+        <input (change)="search(1)" [(ngModel)]="searchTerm" placeholder="search by title or description"/><!-- TODO: delay search until finish typing -->
+      </div>
+
+      <div class="form-group">
+        <span class="left">
+          <label [attr.for]="searchSeries">Filter by Series</label>
+          <select id="searchSeries" [(ngModel)]="searchSeriesId" (ngModelChange)="searchBySeries()">
+            <option *ngFor="let seriesId of allSeriesIds" [value]="seriesId">{{allSeries[seriesId] ? allSeries[seriesId].title : 'No Series'}}</option>
+          </select>
+          <!-- TODO: there should be a way to clear this -->
+        </span>
+
+        <span class="right">
+          <!-- last-modified, published-date, title -->
+          <label [attr.for]="orderBy">Order by</label>
+          <select id="orderBy" [(ngModel)]="allSeries" (ngModelChange)="onChange($event)">
+            <option *ngFor="let series of allSeriesIds" [value]="series">{{series}}</option>
+          </select>
+        </span>
+      </div>
+
+      <div>
+        <button class="tab">Series</button>
+        <button (click)="search(1)" class="tab active" disabled>Stories</button>
+
+        <span class="pages">
+          <button *ngIf="totalCount && pages?.length > 1" (click)="search(1, totalCount)" class="btn-link">View All {{totalCount}} &raquo;</button>
+          <button disabled *ngIf="totalCount && pages?.length === 1" class="btn-link">Viewing {{totalCount > 1 ? 'all ' + totalCount : totalCount}}</button>
+          <!-- How is the user supposed to get back to the paged view? -->
```

Comment:
Do they need to get back to the paged view?  Altering the search should trigger this button to re-enable, right?